docs/data_transfer.md

```diff
@@ -24,19 +24,18 @@ These bottom-up numbers indicate that the majority of emissions from broadband u
 
 ## Power usage by bandwidth
 
-For mobile vs fixed data there is a great [2020 EU study](https://circabc.europa.eu/sd/a/8b7319ba-ce4f-49ea-a6e6-b28df00b20d1/ICT%20impact%20study%20final.pdf) that provides 0.03 kWh/GB for fixed, 0.14 kWh/GB for mobile. The 0.03 kW/GB number aligns with what is in the graphic above (0.03 W/Mbps)?
+For mobile vs fixed data there is a great [2020 EU study](https://circabc.europa.eu/sd/a/8b7319ba-ce4f-49ea-a6e6-b28df00b20d1/ICT%20impact%20study%20final.pdf) that provides 0.03 kWh/GB for fixed, 0.14 kWh/GB for mobile. The 0.03 kWh/GB number aligns with what is in the graphic above (0.03 W/Mbps) at relatively low bandwidth.
```

Comment:
I tried to go to `https://circabc.europa.eu/sd/a/8b7319ba-ce4f-49ea-a6e6-b28df00b20d1/ICT%20impact%20study%20final.pdf` and couldn't resolve that link ? Gave me an error `Failed to load PDF document.
`